### ISSUE TITLE
Using no validation set fails

### ISSUE THREAD

**Issue by ndryden** [LLNL/lbann#107]
Using the ImageNet data reader through prototext with no validation set (i.e. `validation_percent: 0`) results in this error:
`/usr/workspace/wsa/dryden1/lbann/src/data_distributions/partitioned_minibatch.cpp 221 :: generic_data_distribution: number of parallel readers is zero`

We should be able to run with no validation set (if we only care about running on the training+test set and want to use all the training data).

**Comment by bvanessen** [comment]
Fixed in d134b4f942dbe37a108957f0c2b2edc39366cd98
